chicago/permit_cleaning.py

```diff
@@ -221,7 +245,7 @@ def flag_fix_long_fields(df):
 
     for flag_name, column, limit, comment in long_fields_to_flag:
         df[flag_name] = df[column].apply(lambda val: 0 if pd.isna(val) else (1 if len(str(val)) > limit else 0))
-        df["FLAG COMMENTS"] += df[column].apply(lambda val: "" if pd.isna(val) else ("" if len(str(val)) < limit else comment + str(len(str(val)) - limit) + "; "))
+        df["FLAG COMMENTS"] += df[column].apply(lambda val: "" if pd.isna(val) else ("" if len(str(val)) <= limit else comment + str(len(str(val)) - limit) + "; "))
```

Comment:
This is the root cause of https://github.com/ccao-data/extract-permits/issues/16: We were applying the flag to `df[flag_name]` if `len(str(val)) > limit`, but the same flag was being added to `df["FLAG COMMENTS"]` if `not len(str(val)) < limit`, which is equivalent to `len(str(val)) >= limit`.